### ISSUE TITLE
 Clarification of the Resources declaration page

### ISSUE THREAD

**Issue by 7neyugn7** [UQ-RCC/nimrod-portal#11]
[Moved from https://github.com/UQ-RCC/portal-client/issues/1 ]
On the Resources tab,
When the user declares a new resource it might be clearer if we use the term bundle instead of the term batch on that page, provide some clarification and reorganise the resources.

The Resources UI has two sections.

    Currently called "Job Specification"
    Suggest change title to "Nimrod Agent Resources Specification"
    The first section contains the resource request for each agent.
    The ncpus and mem settings are there.
    We should add the walltime to this first section

A mouseover bubble or in the sub-heading it could say something like

    These are the resources that will be allocated per agent

    Currently called "Batch Specification"
    Suggest changing the title to "Configuration of Bundles of Nimrod Agents"
    The second section is about the configuration of agents into bundles.
    Again, if a mouseover bubble or a modification to the subsection heading could indicate

    The bundle size is the number of agents that will be deployed per PBS batch job (the job resources will be multiplied up). A bundle maps onto a single PBS job submission.
    and
    The number of bundles multiplies up the number of PBS job submissions that are made.

So in other words

    agent resources * bundle size -> PBS job footprint (make sure that this fits within the resource footprint of the available nodes)
    number of bundles -> number of PBS jobs submitted

Based on what experiments in the vHPC session

    ncpus and mem request values apply to each agent and is the resource footprint of each deployed agent
    bundle-size is the number of agents deployed per job that the Nimrod Master qsubs
    bundle number is the number of qsubs made

So ncpus=1, mem=3GB combined with
bundle-size=6 and bundle-number=2 and walltime of 3hours gave us
The master job (ncpus=1,mem=2GB and a 24 hour walltime) was deployed
and then the master queued up
2 jobs each with resources of 6 x (ncpus=1,mem=3GB and a 3 hour walltime)
